src/lib/components/Image/Image.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Image.css';
+
+const Image = (props) => {
+  let classArray = [];
+
+  if (props.bordered) {
+    classArray = classArray.concat('p-image--bordered');
```

Comment:
use `.push` instead, `.concat` is to [merge arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat).  We should keep the meaning of the functions even if they give the same result

Reply:
Using `.concat` was a deliberate choice to not mutate the original array, which isn't an issue here but treating arrays as immutable is useful when dealing with state changes and lifecycle events in React (and by extension Redux). You've reminded me about the spread operator though, which achieves the same thing but in a cleaner syntax (in my opinion).